### PR TITLE
tikv: update CDC config

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5961
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1269,6 +1269,25 @@ Configuration items related to BR backup.
...
 + Minimum value: `1`
 
+## cdc <span class="version-mark">New in v4.0.5</span>
+
+Configuration items related to TiCDC capturing changed data.
+
+### `min-ts-interval`
+
++ The interval at which timestamps are resolved.
++ Default value: `"1s"`
```
Comment (ran-huang) [inline tikv-configuration-file.md:1279]: Is the quotation mark correct?

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1269,6 +1269,25 @@ Configuration items related to BR backup.
 + Default value: `MIN(CPU * 0.75, 32)`.
 + Minimum value: `1`
 
+## cdc <span class="version-mark">New in v4.0.5</span>
```
Comment (ran-huang) [inline tikv-configuration-file.md:1272]: This version mark should be removed from master

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1269,6 +1269,25 @@ Configuration items related to BR backup.
...
+
+### `old-value-cache-size` <span class="version-mark">New in v5.0</span>
+
++ The entry number of TiCDC old values cached in memory.
++ Default value: `1024`
+
+### `incremental-scan-speed-limit` <span class="version-mark">New in v5.0</span>
+
++ The maximum speed at which historical data is incrementally scanned.
++ Default value: `"128MB"`, which means 128 MB per second.
```
Comment (ran-huang) [inline tikv-configuration-file.md:1289]: ditto

File: tikv-configuration-file.md (hunk 1 above)
Comment (TomShawn) [reply tikv-configuration-file.md:1279]: I searched similar default values in this doc like `"xs"` and all are wrapped in "".

File: tikv-configuration-file.md (hunk 2 above)
Comment (TomShawn) [reply tikv-configuration-file.md:1272]:
```suggestion
## cdc
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1269,6 +1269,25 @@ Configuration items related to BR backup.
...
+## cdc <span class="version-mark">New in v4.0.5</span>
+
+Configuration items related to TiCDC capturing changed data.
+
+### `min-ts-interval`
+
++ The interval at which timestamps are resolved.
++ Default value: `"1s"`
+
+### `old-value-cache-size` <span class="version-mark">New in v5.0</span>
```
Comment (TomShawn) [inline tikv-configuration-file.md:1281]:
```suggestion
### `old-value-cache-size`
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1269,6 +1269,25 @@ Configuration items related to BR backup.
...
+
++ The entry number of TiCDC old values cached in memory.
++ Default value: `1024`
+
+### `incremental-scan-speed-limit` <span class="version-mark">New in v5.0</span>
```
Comment (TomShawn) [inline tikv-configuration-file.md:1286]:
```suggestion
### `incremental-scan-speed-limit`
```

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -1269,6 +1269,25 @@ Configuration items related to BR backup.
 + Default value: `MIN(CPU * 0.75, 32)`.
 + Minimum value: `1`
 
+## cdc
+
+Configuration items related to TiCDC capturing changed data.
```
Comment (amyangfei) [inline tikv-configuration-file.md:1274]:
```suggestion
Configuration items related to TiCDC.
```
TiCDC is short for `TiDB or TiKV change data capture`

File: tikv-configuration-file.md [hunk 7]
```diff
@@ -1269,6 +1269,25 @@ Configuration items related to BR backup.
...
+
+### `min-ts-interval`
+
++ The interval at which timestamps are resolved.
```
Comment (amyangfei) [inline tikv-configuration-file.md:1278]:
```suggestion
+ The interval at which resolved ts are calculated and forwarded.
```

File: tikv-configuration-file.md (hunk 1 above)
Comment (amyangfei) [reply tikv-configuration-file.md:1279]: wrapped with `""` is correct, which means it is a string

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -1269,6 +1269,25 @@ Configuration items related to BR backup.
 + Default value: `MIN(CPU * 0.75, 32)`.
 + Minimum value: `1`
 
+## cdc
+
+Configuration items related to TiCDC.
+
+### `min-ts-interval`
+
++ The interval that `resolved ts` is calculated and forwarded.
```
Comment (TomShawn) [inline tikv-configuration-file.md:1278]:
```suggestion
+ The interval at which Resolved TS is calculated and forwarded.
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**